apiserver/facades/agent/deployer/register.go

```diff
@@ -43,11 +44,19 @@ func NewDeployerFacade(ctx facade.ModelContext) (*DeployerAPI, error) {
 	}
 
 	controllerConfigGetter := ctx.ServiceFactory().ControllerConfig()
-	secretBackendAdminConfigGetter := secretbackendservice.BackendConfigGetterFunc(
-		ctx.ServiceFactory().SecretBackend(), ctx.ModelUUID())
+	serviceFactory := ctx.ServiceFactory()
```

Comment:
We assign to serviceFactory but below we still use `ctx.ServiceFactory()` in a couple of places